OrbitClientProtos/capture_data.proto

```diff
@@ -74,6 +73,7 @@ message CaptureInfo {
   repeated CallstackInfo callstacks = 6;
   repeated CallstackEvent callstack_events = 7;
   map<uint64, string> key_to_string = 8;
+  map<uint64, FunctionStats> function_stats = 9;
```

Comment:
Is this map enough to reconstruct `repeated FunctionInfo selected_functions = 1;` and don't save this field field anymore?

Reply:
Not 100% sure if address_infos contains the information also for the selected functions. Will have a look into this.

Reply:
Seem to be strange. There is PreSave in Capture, which updates addresses of selected functions in the SamplingProfiler. However, address_infos comes directly from App. I suggest to look further into this in a separate CL and just add comment for now.

Reply:
It's more for future: even if it's so it should/could be in the separate PR

Reply:
As I added a todo above, I'll resolve this for now.